carbonite/src/main/scala/Carboniter.scala

```diff
@@ -0,0 +1,3 @@
+object Carboniter {
```

Comment:
This looks like a stowaway from a related in-progress project

Reply:
ha!  whoops